### DISCUSSION TITLE
请问一下这个右键菜单怎么弄

### DISCUSSION THREAD

**Discussion by kaino3** [hooke007/mpv_PlayKit#520] [Q&A 问答]
看到readme里面这个右键菜单很好看，请问下怎么弄，没找到，感谢各位大大
![image](https://github.com/user-attachments/assets/182a85d5-768e-404a-b72f-18c10f5a57c8)

**Comment by hooke007** [comment]
这个脚本已经被删掉了。相关帖子自行搜索

**Reply by kaino3** [reply]
看到了
感谢老板
